src/modules/scores.js

```diff
@@ -0,0 +1,7 @@
+export default function dynamicScore(userValue, scoreValue) {
```

Comment:
- [ ] As it is a [requirement](https://github.com/microverseinc/curriculum-javascript/blob/main/leaderboard/m2_send_receive_data.md) for this project, " `Use arrow functions instead of the function keyword.` ", please kindly fix that.